state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1773,6 +1783,12 @@ pub mod pallet {
 						);
 
 						dca_state.status = DcaStatus::ChunkScheduled(swap_id);
+						dca_state
+							.network_fee_collected
+							.saturating_accrue(swap.network_fee_taken.unwrap_or_default());
+						dca_state
+							.accumulated_input_amount
+							.saturating_accrue(swap.swap.input_amount);
```

Comment:
As discussed, this needs to be the amout of the stable asset. Should probably change the tests so they would catch this.